### ISSUE TITLE
consider-using-augmented-assign false positive on string formatting

### ISSUE THREAD

**Issue by jolaf** [PyCQA/pylint#8179]
### Bug description

s = "text %s"
s = s % 5

### Command used

```shell
pylint test.py
```

### Pylint output

```shell
************* Module test
test.py:2: [R6104(consider-using-augmented-assign), ] Use '%=' to do an augmented assign directly
```

### Expected behavior

no output

### Pylint version

```shell
pylint 2.16.1
astroid 2.14.1
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0]
```

### OS / Environment

Ubuntu 22.04.1

**Comment by Pierre-Sassoulas** [comment]
One way to fix generically would be to make it configurable, something like an option named ``augmented-assign--recommendation-operators``.
